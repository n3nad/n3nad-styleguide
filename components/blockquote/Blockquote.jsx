import React from 'react'

function Blockquote({ children }) {
  return (
    <blockquote>
      {children}
    </blockquote>
  )
}

export default Blockquote
