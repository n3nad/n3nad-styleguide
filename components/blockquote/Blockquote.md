Text

```jsx
<Blockquote>
  This is blockquote
</Blockquote>
```

