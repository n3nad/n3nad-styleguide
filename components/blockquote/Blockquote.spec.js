
import React from 'react'
import { shallow } from 'enzyme'

import Blockquote from './Blockquote'

describe('Blockquote component', () => {

  it("should render properly", function() {
    const wrapper = shallow(<Blockquote>Some text inside</Blockquote>)

    expect(wrapper).toMatchSnapshot()
  });
})
