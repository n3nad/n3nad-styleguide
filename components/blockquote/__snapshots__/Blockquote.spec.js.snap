// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Blockquote component should render properly 1`] = `ShallowWrapper {}`;
